lib/commands/recordscreen.js

```diff
@@ -263,11 +271,18 @@ commands.startRecordingScreen = async function (options = {}) {
  * @returns {string} Base64-encoded content of the recorded media file if 'remotePath'
  *                   parameter is empty or null or an empty string.
  * @throws {Error} If there was an error while getting the name of a media file
- *                 or the file content cannot be uploaded to the remote location.
+ *                 or the file content cannot be uploaded to the remote location
+ *                 or screen recording is not supported on the device under test.
  */
 commands.stopRecordingScreen = async function (options = {}) {
   const {remotePath, user, pass, method} = options;
 
+  const apiLevel = await this.adb.getApiLevel();
```

Comment:
This validity check could be extracted out, since it is repeated twice (and in the future might change in both places, as different versions of things support different things).

Reply:
fixed